lib/compat/wordpress-6.1/script-loader.php

```diff
@@ -81,6 +98,7 @@ function gutenberg_enqueue_global_styles() {
 // Enqueue global styles, and then block supports styles.
 add_action( 'wp_enqueue_scripts', 'gutenberg_enqueue_global_styles' );
 add_action( 'wp_footer', 'gutenberg_enqueue_global_styles', 1 );
+add_action( 'wp_head', 'gutenberg_enqueue_global_styles_custom_css', 102 );
```

Comment:
Shouldn't this be in the footer, printed _after_ all other global styles?
The way the code is now, we print global styles in the footer (the line above this change), and custom-css in the head
```suggestion
add_action( 'wp_footer', 'gutenberg_enqueue_global_styles_custom_css' );
```

Reply:
@aristath the global styles [only load in the footer for classic themes](https://github.com/WordPress/gutenberg/blob/trunk/lib/compat/wordpress-6.1/script-loader.php#L42) - and I don't think we need to worry about this for global styles custom CSS as it won't be loading in classic themes.